masu/processor/celery/worker.py

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Celery worker module."""
+
+import logging
+
+from celery import Celery
+from flask import Flask
+
+
+def make_celery(app):
```

Comment:
I'm not following why we have a processor and this celery app factory? I think this factory function is all we need? What am I missing?

Reply:
`make_celery()` is straight out of the Celery docs.

I created the `Processor` class for organization purposes. I haven't yet decided which idioms make sense to use, but I'm starting to lean towards more object-oriented. At some point, I think the factory function will move. 

But, the short answer is that the discrepancy is a product of me iterating through the tutorial docs until I'm comfortable enough with what they describe to start thinking about how I want to organize the structures.